nss-day-cohort-56/rare-server-team-1-2
# made admin post approved on creation

# Description

changed create in postviews so that if user is admin the post is approved on creation

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# How Can This Been Tested?
git fetch origin cml-admin-approved
cd  cml-admin-approved
open Rare browser and log in as admin
when you hit create and are redirected to post details the admin button for approving/unapproving should say unapprove indicating it is already approved - 
log in as a non admin and create a post after being redirected go to the post list 
the authors post should not appear because it has not been approved 



# Checklist:
